src/nrtk_explorer/app/transforms.py

```diff
@@ -178,7 +178,7 @@ def delete_meta_state(old_ids, new_ids):
         )
 
         self.server.controller.add("on_server_ready")(self.on_server_ready)
```

Comment:
`self.server.controller.on_server_ready.add(self.on_server_ready)` to be inline with the following line